pkg/services/folder/folderimpl/folder.go

```diff
@@ -202,6 +202,23 @@ func (s *Service) DBMigration(db db.DB) {
 	s.log.Debug("syncing dashboard and folder tables finished")
 }
 
+func (s *Service) CountFoldersInOrg(ctx context.Context, orgID int64) (int64, error) {
+	if s.features.IsEnabledGlobally(featuremgmt.FlagKubernetesClientDashboardsFolders) {
+		resp, err := s.k8sclient.GetStats(ctx, orgID)
```

Comment:
just curious, why not to call `s.unifiedstore.CountInOrg` ?

Reply:
oops! good call - switched it over